lib/cisco_node_utils/cmd_ref/snmpnotification.yaml

```diff
@@ -0,0 +1,11 @@
+# snmpnotification
+---
+enable:
+  config_get: "sh run snmp all | include 'snmp-server enable traps'"
+  config_get_token: '/^(no )?(snmp-server) enable traps <trap_name> ?$/'
+  config_set: "<state> snmp-server enable traps <trap_name>"
+
+notifications:
+  config_get: "sh run snmp all | include 'snmp-server enable traps'"
+  config_get_token: '/.*snmp-server enable traps (.*)?$/'
```

Comment:
This does not look right to me.  You only want to gather trap resources that are enabled right?  If I am reading this right, this will gather enabled and disabled (prefixed with 'no' in the running-config).  I think you wan to anchor like this...

'/^snmp-server enable traps (.*)$/'